docs/src/Dropdown.doc.js

```diff
@@ -231,54 +212,425 @@ card(
+            handleSelect={handleSelect}
+            selected={selected}
+            badgeText="New"
+            option={{ value: "Report a bug", label: "Report a bug" }}
+          />
+          <Dropdown.Item
+            isExternal
+            href="https://help.pinterest.com/en?source=gear_menu_web"
+            option={{ value: "Get help", label: "Get help" }}
+          />
+          <Dropdown.Item
+            isExternal
+            href="https://policy.pinterest.com/en/privacy-policy"
+            option={{ value: "See terms and privacy", label: "See terms and privacy" }}
+          />
+        </Dropdown>
+      )}
+    </Box>
+  );
+}`}
+      />
+      <MainSection.Card
+        cardSize="md"
+        type="don't"
+        description="Avoid adding custom elements within Dropdown. While some custom elements may be technically possible, it is best to avoid customization that becomes difficult to maintain."
+        defaultCode={`
+function CustomContentDropdownExample() {
+  const [open, setOpen] = React.useState(false);
+  const [selected, setSelected] = React.useState(null);
+  const anchorRef = React.useRef(null);
+  const handleSelect = ({item}) => {
+    setSelected(item);
+  };
+
+  return (
+    <Box display="flex" justifyContent="center">
+      <Button
+        accessibilityControls="dont-custom-dropdown-example"
+        accessibilityHaspopup
+        accessibilityExpanded={open}
+        iconEnd="arrow-down"
+        text="Menu"
+        inline
+        ref={anchorRef}
+        selected={open}
+        onClick={ () => setOpen((prevVal) => !prevVal) }
+      />
+      {open && (
+        <Dropdown id="dont-custom-dropdown-example" onSelect={(event, item) => {console.log("Selecting", event, item);}} anchor={anchorRef.current} onDismiss={() => {setOpen(false)}}>
+          <Dropdown.Item
+            handleSelect={handleSelect}
+            selected={selected}
+            option={{ value: "Settings", label: "Settings" }}
+          />
+          <Dropdown.Item
+            handleSelect={handleSelect}
+            selected={selected}
+            badgeText="New"
+            option={{ value: "Report a bug", label: "Report a bug" }}
+          />
+          <Dropdown.Item
+            isExternal
+            href="https://help.pinterest.com/en?source=gear_menu_web"
+            option={{ value: "Get help", label: "Get help" }}
+          >
+            <Icon icon="ad" color="darkGray" accessibilityLabel="Ad"/>
+            <Text>Get help</Text>
+          </Dropdown.Item>
+          <Dropdown.Item
+            isExternal
+            href="https://policy.pinterest.com/en/privacy-policy"
+            option={{ value: "See terms and privacy", label: "See terms and privacy" }}
+          />
+        </Dropdown>
+      )}
+    </Box>
+  );
+}`}
+      />
+    </MainSection.Subsection>
+  </MainSection>,
+);
+
+card(
+  <MainSection name="Accessibility">
+    <MainSection.Subsection
+      title="ARIA attributes"
```

Comment:
we should find a consistent way of description Aria attributes 

From Popover 
![image](https://user-images.githubusercontent.com/10593890/112026465-a2ac4780-8b0c-11eb-9554-3695dd5737b0.png)